hphp/util/etch-helpers.h

```diff
@@ -49,6 +49,30 @@
 #define ETCH_GET_ARG6     /* not used */
 #define ETCH_RET1         %rax
 
+#elif defined(__aarch64__)
+#define CFI(x)          .cfi_##x
+#define CFI2(x, y)      .cfi_##x y
+#define CFI3C(x, y, z)  .cfi_##x y##, z
+#define ETCH_ALIGN16    .align 4
```

Comment:
I think it is better use directly balign or p2align instead of align here.  Then you don't have to worry about figuring out if the operand is a power of 2 or just the byte size.